AndelaOSP/wire
# #167639770 Upgrade outdated packages

#### What does this PR do?
This pull request updates the package dependencies of the application. Some of them we out of date and exposed the application to vulnerabilities and security issues.
#### Description of Task to be completed?
- Upgrading the application and updating the `yarn.lock` file.
 
#### How should this be manually tested?
 - Check out this branch `ch-upgrade-packages-167639770`.
 - Run `yarn install` to re-install all dependencies.
 - Run tests to confirm that there are no breaking changes. 

#### What are the relevant pivotal tracker stories?
[167639770](https://www.pivotaltracker.com/story/show/167639770)
